xml/cap_depl_install_production.xml

```diff
@@ -416,8 +416,18 @@ clusterrolebinding.rbac.authorization.k8s.io "cap:clusterrole" created</screen>
    <para>
     Verify that the new PSPs exist by running the
     <command>kubectl get psp</command> command to list them.
-    Then continue by deploying UAA and SCF.
+    Then continue by deploying UAA and SCF. Ensure that your
+    <filename>scf-config-values.yaml</filename> file specifies
+    the name of your PSP in the <literal>kube:</literal>
+    section. These settings will grant only the
+    <literal>diego-cell</literal>, <literal>nfs-broker</literal>,
+    and <literal>router</literal> roles to be privileged.
```

Comment:
Consider just saying _a limited subset of roles_ rather than explicitly listing them, since we might need to change that list from time to time.

Reply:
Sounds good to me